src/apis/abusiveexperiencereport/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@google/abusiveexperiencereport",
```

Comment:
are we sure that names for all libraries are unique on npm? maybe use some other npm org?

Reply:
They're all scoped under @google, so I'm pretty sure :)